scripts/polkadotjs_endpoints_to_preconfigured.py

```diff
@@ -275,16 +275,18 @@ def process_json_file(file_path):
 
 
 def process_dict_data(file_path, data):
-    if "(SHUTTING DOWN)" not in data.get('name', ''):
+    if "(SHUTTING DOWN)" in data.get('name', '') or "Westend" in data.get('name', ''):
+        pass
+    else:
         os.remove(file_path)
         print(f"Removed file: {file_path}")
 
 
 def process_list_data(file_path, data):
-    shutting_down_items = [entry for entry in data if "(SHUTTING DOWN)" in entry.get('name', '')]
-    if shutting_down_items:
+    skipped_items = [entry for entry in data if "(SHUTTING DOWN)" in entry.get('name', '') or "Westend" in entry.get('name', '')]
```

Comment:
there we can use skip_patterns array, if define it on base level in order to decrease repetition